runtime/onert/api/src/nnfw_api_internal.cc

```diff
@@ -352,7 +352,7 @@ NNFW_STATUS nnfw_session::load_model_from_nnpackage(const char *package_dir)
     }
     _nnpkg = std::make_shared<onert::ir::NNPkg>();
     auto num_models = models.size();
-    if (num_models >= static_cast<uint16_t>(num_models))
+    if (num_models == 0 || (num_models - 1) > onert::ir::ModelIndex::max())
```

Comment:
@hseok-oh Why `-1` is necessary? I think `num_models` == `ModelIndex::max()` is okay.

Reply:
`num_models` is size, but `max()` is index. So it needs `-1` for comparison.